### PR TITLE
Woo Tailored Onboarding: Add new domain/plans steps to ecommerce flow

### PR DESCRIPTION
**Pull request by nelsonec87**
## This is work in progress!

* This PR builds on top of https://github.com/Automattic/wp-calypso/pull/68722 
* Adds Domains & Plan steps to the new `ecommerce` flow

#### Testing Instructions
* Make sure the flag `signup/tailored-ecommerce` is enabled in your environment
* Access `/setup?flow=ecommerce` and you'll be redirected to `/setup/domains?flow=ecommerce`
* Pick a domain

* You'll be redirected to `/setup/plans?flow=ecommerce`
* Pick a Plan

* If you picked a paid domain/plan you'll go to checkout

* After checkout, you'll be redirected to the `intro` step and your site will already be created

* Click create your store and you'll be redirected to `storeProfiler`

* Click Continue and you'll be redirected to `designCarousel` step

* Choose a theme and click Continue.
* You should be redirected to `/home/<new-site-slug>`. 
**The Site should have the plan/domain/theme you chose in the flow.**

### CODE REVIEW

File: client/my-sites/plans-features-main/index.jsx [hunk 1]
```diff
@@ -118,6 +118,8 @@ export class PlansFeaturesMain extends Component {
 			siteId,
 			plansWithScroll,
 			isReskinned,
+			isFAQCondensedExperiment,
```
Comment (niranjan-uma-shankar) [inline client/my-sites/plans-features-main/index.jsx:121]: Seems to be a remnant from a rebase, this flag is not required.

File: client/my-sites/plans-features-main/index.jsx [hunk 2]
```diff
@@ -162,6 +164,8 @@ export class PlansFeaturesMain extends Component {
 					} ) }
 					siteId={ siteId }
 					isReskinned={ isReskinned }
+					isFAQCondensedExperiment={ isFAQCondensedExperiment }
```
Comment (niranjan-uma-shankar) [inline client/my-sites/plans-features-main/index.jsx:167]: Ditto, flag not used and can be removed.

**Review by niranjan-uma-shankar — COMMENTED** [review]
A few notes:

* The calypso.live URL didn't work with `/setup?flow=ecommerce`, but it did after I added `/setup?flow=ecommerce&flags=signup/tailored-ecommerce`. 

* The flow doesn't work if I'm logged out and enter the setup flow. After a paid plan, I'm taken to the user step and I never reach the checkout page. 

* The plans step width (overall and individual plan column) seems to be constricted and smaller than the regular plans step width in the main signup flow. 

* The "Choose design" step is blank, when I enter the flow either logged in or logged out:

Screenshot 2022-11-03 at 10 57 18 AM

**Comment by nelsonec87** [comment]
Closing in favor of more up to date https://github.com/Automattic/wp-calypso/pull/69974

**Pull request closed without merging**